backend/routes/api.php

```diff
@@ -18,12 +18,14 @@
 
 Route::post('/register', [StudentController::class, 'store']);
 Route::post('/login', [StudentController::class, 'login']);
+Route::get('/students/{id}', [StudentController::class, 'show']);
 
 // dashboard
 Route::get('/dashboard/{id}', [DashboardController::class, 'index']);
 
 // protected routes
 Route::group(['middleware' => ['auth:sanctum']], function () {
+  Route::put('/students/update/{id}', [StudentController::class, 'update']);
```

Comment:
For this one no need to have the id of the user since you passed the bearer token on the request. Since we have middleware auth:sanctum for this and if the configuration is correct we should be be able to get user via `auth()->user()`

Reply:
you can also apply this change in your FE route (to make it unified)

Reply:
Thank you Sir. I'll update this one.